tests/modules/nf-core/chopper/test.yml

```diff
@@ -0,0 +1,17 @@
+- name: "chopper"
+  command: nextflow run ./tests/modules/nf-core/chopper -entry test_chopper -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/chopper/nextflow.config
+  tags:
+    - "chopper"
+  files:
+    - path: "output/chopper/test.filtered.fastq.gz"
+      contains:
+        [
+          "@2109d790-67ec-4fd1-8931-6c7e61908ff3 runid=97ca62ca093ff43533aa34c38a10b1d6325e7e7b read=52274 ch=243 start_time=2021-02-05T23:27:30Z flow_cell_id=FAP51364 protocol_group_id=data sample_id=RN20097 barcode=barcode01 barcode_alias=barcode01",
+          "ATTGTACTTCGTTCAGTTACGTATTGCTAAGGTTAACACAAAGACACCGACAACTTTCTTCAGCACCTAAACATGGAGGAGGTGTTGCAGTGAGCCGAAACCATGCCATTGCACTCCAGCCTGGGTGACAGAGTGAGACTCCTTCTGAAAAAAAAAAAAAAAAAAAAGACACAGGTACCATCTCTAAAGCCCTCCAGTTTCCTCTCTCCCACATCCCAACCGTCTCAAACTCAGAGTGCTGAAGAAAGTTGTCAGTGTCTTGTGTTAACCCTTAACTGTACGTAATGACA",
+          "%')%%%113(47;84=A;111857-66-=><7ABCD;B>JLHFAE@>5366<=9?@>7;'7=88::;HGGBC<<:47-.+54<A>JB=<>?@;=<8359854340;9348=?5=?;<@<*7;DG;<=;8$$$'27/-05531254458,????ABD@;63/,*(&%$)/.,..64;9:;>?<,2*/2.+4:+67;9:;?=2)778:BECB?=BE@?@0133995;-$%$$'&+,&599=;?BA7>;=9:936.*031/***)%%)./.670/33)%%#*',**21$%(,(",
+          "@1377f442-7e38-4c3f-aacb-e8049ad884fe runid=97ca62ca093ff43533aa34c38a10b1d6325e7e7b read=58589 ch=376 start_time=2021-02-05T23:19:54Z flow_cell_id=FAP51364 protocol_group_id=data sample_id=RN20097 barcode=barcode01 barcode_alias=barcode01",
+          "ATTGTACTTCGTTCAGTTACATGTGCTAGGTTAACACAAAGACACCGACAACTTTCTTCAGCACCTGAACAAAGACCATTAAATTACTCTGGAAACGGAATGAAGAATGAAGGAATGAAGCATGGCCAGGACTCAAGGACCAAAGAGGACAGGGGTGCGAGGTGAGCAGCAGATGGAGGCAGGGCCAGATCACGTAGCGGGTGGAGGCCGCCTTAGCCACACAGGTGTTCTGGGAAGAGCAATGGTGCTGAGTAAACTAACAGGCAGTGTGAGTGGTGGTAAAGGCATCAAGAGTGCAGTTGTCAGGGAAGGACATAAGATGATAGCCCAGGTGCTGAAGAAAGTTGTCGGTGTCTTTGTGTTAACCTTATGCTCTATG",
+          "$#%&(&4573489+&1*/*&$$$)+63332CBA:?::@@3*69:<.+-/-/37:>=B7:B?@B;,<<B@>AD@:?76***)*,-/56879+(642)+489:<@@::>B841%%6=;:@@:AAA@>;96>BB8?77:8;>@?;@5168:<?7=136<>86*5:=5.)'12@;AB@:,*,/,-78<ACA;:B1002410%$(45:3*'59<002()++++,..136=?@894412DLG=A>A>?A<070$()+0455733::85<5;;;@DC><7?@9=?>:>>:,*;??;-*((/6>B?CDGA;/+//;673;92--6303.**0,/;=777<=97924,.446349997,/79:7;<914.1422%''%$&#'((')($",
+        ]
+    - path: "output/chopper/versions.yml"
+      md5sum: 49a1f5cfbcbe34de24138a9e7eb1e24f
```

Comment:
```suggestion
```

Not worth it as then have to re-gen for every version bump

Reply:
should be fixed in new commit